st2actions/st2actions/cmd/scheduler.py

```diff
@@ -8,7 +8,10 @@
 import signal
 import sys
 
-from st2actions.scheduler import scheduler
+from st2actions.scheduler import (
+    handler as scheduler_handler,
+    entrypoint as scheduler_entrypoint
+)
```

Comment:
Yeah, let's please use a consistent import style - I use one style (which was used since the beginning), then @m4dcoder started to use another and now yet another :P